<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="Main_Menu">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>951</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="login_page">
     <widget class="QLabel" name="login_title">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#00aa00;&quot;&gt;LOGIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="userNameLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userNameLineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="loginEnterButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ENTER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginConctButt">
      <property name="geometry">
       <rect>
        <x>744</x>
        <y>20</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CONTACT US</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginRevButt">
      <property name="geometry">
       <rect>
        <x>744</x>
        <y>60</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>LEAVE A REVIEW</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginHelpButt">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>570</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>HELP</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="admi_page">
     <widget class="QPushButton" name="adminPageBackButt">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>570</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLabel" name="admin_title">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#00aa00;&quot;&gt;ADMIN PAGE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adminHelpButt">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>570</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>HELP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adminContButt">
      <property name="geometry">
       <rect>
        <x>744</x>
        <y>20</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CONTACT US</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="Admin_Pages">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>881</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="main_admin_page">
       <widget class="QPushButton" name="addShapeButt">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>50</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ADD SHAPE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="remShapeButt">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>80</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>REMOVE SHAPE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="editShapeButt">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>110</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>EDIT SHAPE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="viewShapeButt">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>140</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>VIEW SHAPE</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="add_page">
       <widget class="QPushButton" name="back2AdminButt">
        <property name="geometry">
         <rect>
          <x>774</x>
          <y>460</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>BACK TO ADMIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="addShapeLabel">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>40</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;ADD SHAPE (or text)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addFromFileButton">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>80</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add From File</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="remove_page">
       <widget class="QPushButton" name="back2AdminButt_2">
        <property name="geometry">
         <rect>
          <x>774</x>
          <y>460</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>BACK TO ADMIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="addShapeLabel_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;REMOVE SHAPE (or text)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="edit_page">
       <widget class="QPushButton" name="back2AdminButt_3">
        <property name="geometry">
         <rect>
          <x>774</x>
          <y>460</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>BACK TO ADMIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="addShapeLabel_3">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>30</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;EDIT SHAPE (or text)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="view_basic_page">
     <widget class="QPushButton" name="basicPagebackButt">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>590</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLabel" name="viewPageTitle">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#00aa00;&quot;&gt;VIEWING PAGE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewHelpButt">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>570</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>HELP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewContButt">
      <property name="geometry">
       <rect>
        <x>744</x>
        <y>20</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CONTACT US</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>891</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>AREA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PERIMETER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Pick a list medthod :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reviews_page">
     <widget class="QPushButton" name="reviewPageBackButt">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>590</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLabel" name="reviewPageTitle">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#00aa00;&quot;&gt;REVIEWS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reviewHelpButt">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>570</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>HELP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reviewContButt">
      <property name="geometry">
       <rect>
        <x>744</x>
        <y>20</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CONTACT US</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="help_page">
     <widget class="QPushButton" name="helpBackButt">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>590</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLabel" name="helpTitle">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#00aa00;&quot;&gt;HELP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="helpContButt">
      <property name="geometry">
       <rect>
        <x>744</x>
        <y>20</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CONTACT US</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="help_pages">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>881</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="helpMainPage"/>
      <widget class="QWidget" name="how2_add"/>
      <widget class="QWidget" name="how2_rem"/>
      <widget class="QWidget" name="how2_edit"/>
      <widget class="QWidget" name="how2_view"/>
     </widget>
    </widget>
    <widget class="QWidget" name="contact_page">
     <widget class="QPushButton" name="contactBackButt">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>590</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLabel" name="contactPageTitle">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#00aa00;&quot;&gt;CONTACT US&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="contactInfo">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>421</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;Girls Rule Co.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Address:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Girls Only, &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;suite #1C&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Petaluma, CA 94954&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Email:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;kedc@gmail.com&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Phone:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;(707) 336-2113&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;FAX:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;(999) 667-3322&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="contactHelpButt">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>570</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>HELP</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
